src/dotnet/Audio.Service/StreamServerBase.cs

```diff
@@ -7,8 +7,8 @@ public abstract class StreamServerBase<TItem> : IDisposable
 
     protected int StreamBufferSize { get; init; } = 64;
     protected TimeSpan MaxStreamDuration { get; init; } = TimeSpan.FromSeconds(600);
-    protected TimeSpan ReadStreamWaitDuration { get; init; } = TimeSpan.FromSeconds(2);
-    protected TimeSpan ReadStreamExpiration { get; init; } = TimeSpan.FromSeconds(5);
+    protected TimeSpan ReadStreamWaitDuration { get; init; } = TimeSpan.FromSeconds(10);
```

Comment:
Why these are tweaked? IMO such things should be tweaked only if we see real problems with the old settings.

2s wait time is there, coz it returns "no stream" only after spending this wait time. So if client sees the transcript w/ StreamId & hits the server, it shouldn't wait a lot to discover there is no stream, I guess.

Reply:
Let's discuss this as well. Maybe 5s is fine assuming ReadStreamExpiration is 15s.

Reply:
in tests we have   ```
523 info: ActualChat.Audio.AudioStreamProxy[0]
      Write(audio #01GPJD65Q8QDN211QFCCQ1R7DH-0000): fallback to the local server
```
and ```
 3058 info: ActualChat.Audio.AudioStreamProxy[0]
      Read(audio #01GPJD65Q8QDN211QFCCQ1R7DH-0000): fallback to the local server
```
so 2 sec is not enough

Reply:
```
  531 info: Stl.Fusion.Operations.Internal.CompletionProducer[0]
      Local operation completion succeeded. Agent: 'LEGION-jQBv1ZSkOkOr05O8Ijswkg-2', Command: UpsertEntryCommand 
```
Entry has been created in 10 ms, but our entry reader hasn't reacted during next 2500 ms

Reply:
At least in tests

Reply:
I have reverted back those timings  - we just had a bug with test itself